contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,61 @@
+# See readme.md for instructions on running this code.
+from google import search
+
+def get_google_result(original_content):
+    original_content = original_content.split(' ', 1)[1]
+    try:
+        urls = search(original_content, stop=20)
+    except:
+        return "Error: search failed."
+
+    if not urls:
+        return "No URLs returned by google."
+
+    urls = list(urls)
+    urls = urls[0]
+
+    return 'Success: ' + str(urls)
+
+class GoogleSearchHandler(object):
+    '''
+    This plugin allows users to enter a search
+    term in zulip and get the top URL sent back
+    to private messages.It looks for messages
+    starting with @google.
+    '''
+
+    def usage(self):
+        return '''
+            This plugin will allow users to search
+            for a given search term on google from
+            zulip. Users should preface messages with
+            @google.
+            '''
+
+    def triage_message(self, message):
+        # return True if we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting google
+        # search links.
+        if message['display_recipient'] == 'google':
+            return False
+        is_google_search = (original_content.startswith('@google'))
```

Comment:
```return is_google_search``` and ```return original_content.startswith('@google')``` do exactly the same thing, so implementing ```return original_content.startswith('@google')``` has no purpose. Unless there's something else it could be used for???